packages/react-dom/src/events/BrowserEventConstants.js

```diff
@@ -43,48 +40,29 @@ const topLevelTypes = {
   topDragOver: 'dragover',
   topDragStart: 'dragstart',
   topDrop: 'drop',
-  topDurationChange: 'durationchange',
-  topEmptied: 'emptied',
-  topEncrypted: 'encrypted',
-  topEnded: 'ended',
-  topError: 'error',
```

Comment:
This one also fires for JS errors and images (not just media) but AFAIK it doesn't truly bubble either way.

Reply:
Correct. It does not fire for images, iframes, videos, or audio tags.